helm/values-uat.yaml

```diff
@@ -138,21 +138,17 @@ microservice-chart:
                 values:
                   - user
   canaryDelivery:
-    create: false
+    create: true
     ingress:
-      create: true
-      canary:
-        type: header
-        headerName: X-Canary
-        headerValue: canary
-        weightPercent: 0
-    service:
-      create: true
-    deployment:
-      create: true
-      image:
-        repository: ghcr.io/pagopa/pagopa-debt-position
-        tag: "0.9.0"
-        pullPolicy: Always
-      envConfig: {}
-      envSecret: {}
+      type: header
+      headerName: X-Canary
+      headerValue: canary
+      weightPercent: 0
+    image:
+      repository: ghcr.io/pagopa/pagopa-debt-position
+      tag: "0.11.63-3-hikari"
+      pullPolicy: Always
+    envConfig:
+      SPRING_DATASOURCE_URL: "jdbc:postgresql://gpd-db.u.internal.postgresql.pagopa.it:5432/apd?sslmode=require&prepareThreshold=0&tcpKeepAlive=true"
```

Comment:
@cap-ang il `tcpKeepAlive` è settato per DEV e UAT, ma non in PRODUZIONE. Sono configurazione volute?

Reply:
> The keepalive mechanism above is ideal for long-standing idle connections [...] Unless you know you'll have long-lived idle connections, and that your backend (or network equipment) will interfere with these connections, it's better to leave this off.
from https://www.npgsql.org/doc/keepalive.html

Based on this doc, I'd set it to false (default value), agree?